src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,92 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int WORK_DATE = 0;
```

Comment:
0 is not a **date**
0 is the **index** of the date in the array
so, the name should be **WORK_DATE_INDEX**